packages/runtime/src/lib/constants.ts

```diff
@@ -8,36 +8,51 @@ export const MAX_CONCAT_SIZE = 4096;
 export const SSC_BYTES = 1; // 1 represents type 'byte' in stateful key-value pair
 export const SSC_UINT = 2; // 2 represents type 'uint' in stateful key-value pair
 export const ALGORAND_MIN_TX_FEE = 1000;
+
+// values taken from https://github.com/algorand/go-algorand/find/master#L689
```

Comment:
Could you provide a link to a source file?

Reply:
Yes, sure